data/Efficiencies/Electron_HLT_DoubleEleLegHigPt_2016_cut_WP_Tight80X_full2016.json

```diff
@@ -0,0 +1,1018 @@
+{
+  "variables": [
+    "Eta", 
+    "Pt"
+  ], 
+  "error_type": "absolute", 
+  "data": [
+    {
+      "bin": [
+        -2.5, 
+        -2.1
+      ], 
+      "values": [
+        {
+          "bin": [
+            0.0, 
+            23.0
+          ], 
+          "error_low": 0.0, 
+          "value": 0.025, 
+          "error_high": 0.0
+        }, 
+        {
+          "bin": [
+            23.0, 
+            24.0
+          ], 
+          "error_low": 0.001, 
+          "value": 0.361, 
+          "error_high": 0.001
+        }, 
+        {
+          "bin": [
+            24.0, 
+            25.0
+          ], 
+          "error_low": 0.001, 
+          "value": 0.643, 
+          "error_high": 0.001
+        }, 
+        {
+          "bin": [
+            25.0, 
+            26.0
+          ], 
+          "error_low": 0.001, 
+          "value": 0.843, 
+          "error_high": 0.001
+        }, 
+        {
+          "bin": [
+            26.0, 
+            30.0
+          ], 
+          "error_low": 0.0, 
+          "value": 0.938, 
+          "error_high": 0.0
+        }, 
+        {
+          "bin": [
+            30.0, 
+            35.0
+          ], 
+          "error_low": 0.0, 
+          "value": 0.965, 
+          "error_high": 0.0
+        }, 
+        {
+          "bin": [
+            35.0, 
+            40.0
+          ], 
+          "error_low": 0.0, 
+          "value": 0.971, 
+          "error_high": 0.0
+        }, 
+        {
+          "bin": [
+            40.0, 
+            45.0
+          ], 
+          "error_low": 0.0, 
+          "value": 0.976, 
```

Comment:
are you sure all the errors were picked up from the file correctly ? there are a bunch of bins with zero error...

Reply:
Yes, but that's already there in the txt files...

Reply:
boh... ok then. could you also add the original txt files for reference ? so that we know in a couple of months were exactly we took these from ?

Reply:
yes sure

Reply:
done ;)